### PR TITLE
refactored string resource ids

### PR DESCRIPTION
**Pull request by Jaakik**
- Refactored String Resource ids as such : FragmentName_descriptionView
- Removed Unused Resources

### CODE REVIEW

**Review by martin-lenweiter — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -8,72 +8,53 @@
...
-    - Once you have finished everything you wanted to do, you can click on the "Save data" button to
-    save your modifications.</string>
-    <string name="displayImagesetInfo">
-        General information :\n\n
-        - Here are displayed all the pictures of the selected category.\n\n
-        - A representative picture is a picture that is used to represent the category in the dataset.
-        Such a picture is not part of the pictures of the category.\n\n
-        Possible actions :\n\n
-        - To delete pictures : click longer on the picture you want to delete, select all the pictures to delete and press the trashcan icon.\n\n
-        - To set an picture as representative of the category : click longer on the picture and select the star icon.</string>
```
Comment (bbjubjub2494) [inline app/src/main/res/values/strings.xml:56]: where did those go?

Comment (Jaakik) [reply app/src/main/res/values/strings.xml:56]: I just removed these when resolving the conflicts, you can see those same resources refactored just above

Comment (bbjubjub2494) [reply app/src/main/res/values/strings.xml:56]: Ok found it. It's different than the original tho, why is that? And where is the information on deleting a category and renaming a dataset?

Comment (bbjubjub2494) [reply app/src/main/res/values/strings.xml:56]: Looking at the view in context, these instructions seem redundant. Is that the idea?

Comment (Jaakik) [reply app/src/main/res/values/strings.xml:56]: Yes that was the idea initially but I think it's Ok to leave them for first time users so I ll put them back -- and readd the info that I mistakenly removed

**Review by bbjubjub2494 — APPROVED** [review]

**Pull request merged**